public/app/percona/shared/websockets/notifications.ts

```diff
@@ -0,0 +1,32 @@
+import { Severity } from './../core/types';
+import { AppEvents } from '@grafana/data';
+import appEvents from 'app/core/app_events';
+import { CheckStreamData } from './notifications.types';
+import { Messages } from './notifications.messages';
+
+export const checkNotification = (data: CheckStreamData) => {
+  data.result.topic === 'checks' &&
+    data.result.results?.forEach((value) => {
+      const message = `${value.summary}/${value.service_name}: ${value.alert_message}`;
+
+      if (!value.alert_message && value.service_name) {
+        appEvents.emit(AppEvents.alertSuccess, [`${value.summary}/${value.service_name}: ${Messages.noProblemFound}`]);
+        return;
+      }
+
+      switch (Severity[value.severity]) {
+        case Severity.SEVERITY_ALERT ||
```

Comment:
Not sure this syntax works as we expect it to.
Usually it's done like this:

```
case Severity.SEVERITY_ALERT:
case Severity.SEVERITY_CRITICAL:
case Severity.SEVERITY_ERROR:
case Severity.SEVERITY_EMERGENCY:

```

Reply:
I think that in javascript you can use OR statement as case value, but I changed it to be sure